src/main/java/frc/robot/Robot.java

```diff
@@ -137,8 +138,12 @@ public void autonomousInit() {
       new Turn(m_hardware, m_robotDrive, turnDegrees),
       new Drive(m_hardware, m_robotDrive, inches(starToMiddle)),
       new Turn(m_hardware, m_robotDrive, turnDegrees),
-      new Drive(m_hardware, m_robotDrive, inches(backHome))
+      new Drive(m_hardware, m_robotDrive, inches(backHome)),
+      new Turn(m_hardware, m_robotDrive, turnDegrees),
+      new Turn(m_hardware, m_robotDrive, turnDegrees),
+      new Turn(m_hardware, m_robotDrive, turnDegrees)
       );
+      //somehow I changed something, distances seem to have been changed.
```

Comment:
Can you clarify what this means?

Reply:
I copy and pasted the code used for the border from the project where it worked, but it seems to have caused the robot to not drive as far as it should...

Reply:
Cool. Let's figure that part out next.